tests/imgls.pl

```diff
@@ -166,35 +433,129 @@ sub get_image_bytes {
     return $filebytes;
 }
 
-sub get_cmd_output {
-    my $prog = shift;
+sub ls_sort {
+    return if ! (@_ or scalar @_ > 1);
 
-    use English;
-    my ($ret, $pid);
-    die "Can't fork: $!" unless defined($pid = open(KID, "-|"));
-    if ($pid) {           # parent
-        $ret .= $_       while <KID>;
-        close KID;
-        return $ret;
+    # XXX t and r issue and y
```

Comment:
What's this?

Reply:
Another reminder for me to support -y and env LS_SAMESORT.  Done.